app/models/event-sub-topic.js

```diff
@@ -6,6 +6,7 @@ export default ModelBase.extend({
   name : attr('string'),
   slug : attr('string'),
 
-  topic  : belongsTo('event-topic'),
-  events : hasMany('event')
+  topic       : belongsTo('event-topic'),
+  events      : hasMany('event'),
+  placeholder : belongsTo('custom-placeholder')
```

Comment:
Ahh yes, It will be expecting `placeholder`. Thanks for pointing it out. Nice catch! :)